BitwardenShared/UI/Tools/Send/Send/SendList/SendListView.swift

```diff
@@ -98,7 +103,9 @@ private struct MainSendListView: View {
                     )
                 }
             }
-            .padding(16)
+            .padding(.top, 16)
+            .padding(.horizontal, 16)
+            .padding(.bottom, FloatingActionButton.bottomOffsetPadding)
```

Comment:
🎨 I noticed that sometimes you remove the original bottom padding and separate it out like this so as to not add extra bottom padding and sometimes you just add bottom padding for the FAB.

https://github.com/bitwarden/ios/blob/34a1278a4331ddb38f76ccb7cd5a9ca110d5396c/BitwardenShared/UI/Vault/Vault/VaultItemSelection/VaultItemSelectionView.swift#L189-L190

I think the scroll view modifier is going to also add extra bottom padding. We should be consistent if we can. If you removed the extra 16pts of padding in `FloatingActionButton.bottomOffsetPadding`, could we just add the additional padding for the FAB without having to specify the top and horizontal padding separately? Like what exists in `VaultItemSelectionView.swift`.